vispy/visuals/tests/test_arrows.py

```diff
@@ -0,0 +1,108 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2015, Vispy Development Team.
+# Distributed under the (new) BSD License. See LICENSE.txt for more info.
+
+import numpy as np
+
+from vispy.visuals.line.arrow import ARROW_TYPES
+from vispy.scene import visuals, transforms
+from vispy.testing import (requires_application, TestingCanvas,
+                           run_tests_if_main, assert_raises)
+from vispy.testing.image_tester import assert_image_approved
+
+
+vertices = np.array([
+    [25, 25],
+    [25, 75],
+    [50, 25],
+    [50, 75],
+    [75, 25],
+    [75, 75]
+]).astype('f32')
```

Comment:
Cleaner to do `], dtype=np.float32)` here instead of `]).astype('f32')`. It also avoids a warning for old `numpy`.